src/Constants.js

```diff
@@ -1,90 +1,95 @@
- * @property {string} [noWinner='Giveaway cancelled, no valid participations.'] Sent in the channel if there's no valid winner for the giveaway.
- * @property {string} [winners='winner(s)'] Displayed next to the embed footer, used to display the number of winners of the giveaways.
+ * @property {string} [giveaway='🎉🎉 **GIVEAWAY** 🎉🎉'] Displayed above the giveaway embed when the giveaway is running.
+ * @property {string} [giveawayEnded='🎉🎉 **GIVEAWAY ENDED** 🎉🎉'] Displayed above the giveaway embed when the giveaway has ended.
+ * @property {string} [inviteToParticipate='React with 🎉 to participate!'] Displayed in the giveaway embed. Invite people to react to the giveaway.
+ * @property {string|MessageObject} [winMessage='Congratulations, {winners}! You won **{this.prize}**!\n{this.messageURL}'] Sent in the channel when the giveaway is ended. "{winners}" will be replaced automatically with the mentions of the giveaway winners.
+ * @property {string} [drawing='Drawing: {timestamp}'] Displayed below "inviteToParticipate" in the giveaway embed. "{timestamp}" will be replaced automatically with the time remaining.
+ * @property {string} [dropMessage='Be the first to react with 🎉'] Displayed in the giveaway embed for drop giveaways.
+ * @property {string|EmbedFooterObject} [embedFooter='{this.winnerCount} winner(s) • Powered by the discord-giveaways package'] The footer of the giveaway embed.
+ * @property {string} [noWinner='Giveaway cancelled, no valid participations.'] Displayed in the giveaway embed when there is no valid winner for the giveaway.
+ * @property {string} [winners='Winner(s):'] Displayed in the giveaway embed before the winners.
  * @property {string} [endedAt='Ended at'] Displayed next to the embed footer, used to display the giveaway end date.
- * @property {string} [hostedBy='Hosted by: {user}'] Below the inviteToParticipate message, in the description of the embed.
- * @property {Object} [units]
- * @property {string} [units.seconds='seconds'] The name of the 'seconds' units
- * @property {string} [units.minutes='minutes'] The name of the 'minutes' units
- * @property {string} [units.hours='hours'] The name of the 'hours' units
- * @property {string} [units.days='days'] The name of the 'days' units
- * @property {Boolean} [units.pluralS='false'] Whether to force removing the "S" which marks the plural when the value is lower than 2
+ * @property {string} [hostedBy='Hosted by: {this.hostedBy}'] Below the "inviteToParticipate" message, in the description of the embed.
  */
-exports.GiveawayMessages = {};
+exports.GiveawayMessages = {
+    giveaway: '🎉🎉 **GIVEAWAY** 🎉🎉',
+    giveawayEnded: '🎉🎉 **GIVEAWAY ENDED** 🎉🎉',
+    inviteToParticipate: 'React with 🎉 to participate!',
+    winMessage: 'Congratulations, {winners}! You won **{this.prize}**!\n{this.messageURL}',
+    drawing: 'Drawing: {timestamp}',
+    dropMessage: 'Be the first to react with 🎉 !',
```

Comment:
would it maybe be a better idea to make a `dropOptions` object? like for pause and lastChance?
so that the boolean and the message are in 1 place.
and also cause it's more like a "external" feature, splitted from the normal gaws?

Reply:
Or on the other side, could it be it gets not displayed, because there is the timestamp?
but some people maybe oversee that, hmm, idk?

Reply:
+ it would "save" space in the storage

Reply:
there is only one extra prop (there is many other props with pause/last chance) so I think we can let it here, it does not really matter

Reply:
alright

Reply:
Other thing: should the default message be adjusted to work for multiple winners too?
Cause currently it's "be the first", which might imply, by default, that there is only 1 winner.
_if you oversee the footer_

Reply:
Drops are 99% of the time 1 winner. Like you're in a spammed chat and the first to react gains the nitro. And it's easier to understand. People can still check the docs to see if they can allow multiple winners

Reply:
alright